### PR TITLE
Facilities listing in search

### PR DESCRIPTION
**Pull request by DanielJackson-Oslo**
Makes sure the list of spaces show which of the filtered facilities it matches.

image

Also fixes some bugs in the filter, where no filters other than Overnatting could be set.

Exposes a new bug in the NSR imports, which is that we haven't run aggregated search and therefore have nil values for all experiences. Added a comment to that one in the relevant PR.

### CODE REVIEW

File: app/views/spaces/index/_space_facility_listing.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<ul class="grid grid-cols-2 gap-2 mt-2">
+  <% filtered_facilities.each do |filtered_facility| %>
+    <% facility = filtered_facility %>
```
Comment (mathias234) [inline app/views/spaces/index/_space_facility_listing.html.erb:3]: Dette ser litt unødvendig ut, hvorfor ikke bare kalle den facility i utgangpunktet?

**Review by mathias234 — COMMENTED** [review]
Kult! fungerer bra kun en liten kommentar

File: app/views/spaces/index/_space_facility_listing.html.erb (hunk 1 above)
Comment (DanielJackson-Oslo) [reply app/views/spaces/index/_space_facility_listing.html.erb:3]: Haha! Godt spottet. Jeg hadde noe fancy schamny logikk der en gang, men den er borte nå. Fikser!

Comment (DanielJackson-Oslo) [reply app/views/spaces/index/_space_facility_listing.html.erb]: Fixed!

**Review by mathias234 — APPROVED** [review]

**Pull request merged**